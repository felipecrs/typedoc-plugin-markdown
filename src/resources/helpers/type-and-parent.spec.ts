import * as Handlebars from 'handlebars';

import { setup } from '../../../test/utils';

describe(`[helpers] typeAndParent`, () => {
  let project: any;

  beforeAll(() => {
    ({ project } = setup());
  });

  test(`should compile`, () => {
    const data = project.findReflectionByName('BaseClass');
    const result = Handlebars.helpers.typeAndParent.call(data.children[3].implementationOf);
    expect(result).toMatchSnapshot();
  });
});
