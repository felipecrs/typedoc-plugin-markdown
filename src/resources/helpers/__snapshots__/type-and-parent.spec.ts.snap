// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[helpers] typeAndParent should compile 1`] = `"[INameInterface](../interfaces/_classes_.inameinterface.md).[name](../interfaces/_classes_.inameinterface.md#name)"`;
